src/routes/cache-hooks/guards/pre-execution.guard.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  CanActivate,
+  ExecutionContext,
+  Inject,
+  Injectable,
+} from '@nestjs/common';
+import { ILoggingService, LoggingService } from '@/logging/logging.interface';
+import { formatRouteLogMessage } from '@/logging/utils';
+
+/**
+ * The PreExecutionLogGuard guard outputs a log line containing the request data.
+ */
+@Injectable()
+export class PreExecutionLogGuard implements CanActivate {
+  private readonly PRE_EXECUTION_LOGGING_DETAIL = 'pre-execution-log';
+  private readonly PRE_EXECUTION_LOGGING_STATUS = 202;
```

Comment:
These can be `static`.

Reply:
Changed [5c9cfb7](https://github.com/safe-global/safe-client-gateway/pull/1120/commits/5c9cfb7870538e0177c956a10ef91b3e7b9b620f)